listener/common_test.go

```diff
@@ -70,10 +70,26 @@ func getOrCreateTestAccount(t *testing.T) int {
 
 // nolint: unparam
 func createTestUploadEvent(inventoryID string, packages bool) HostEgressEvent {
+	ns := "insights"
+	v1 := "prod"
 	ev := HostEgressEvent{
 		Type:             "created",
 		PlatformMetadata: nil,
-		Host:             Host{ID: inventoryID, Account: inventoryID},
+		Host: Host{
+			ID:      inventoryID,
+			Account: inventoryID,
+			Tags: []inventory.StructuredTag{
+				{
+					Key:       "env",
```

Comment:
Isn't better to use testing values "key1, key2, ns1, ns2, val1, val2" to easy distinguish it in the tests?